packages/walletkit/package.json

```diff
@@ -30,8 +30,8 @@
     "eth-lib": "^0.2.8",
     "lodash": "^4.17.14",
     "sleep-promise": "^8.0.1",
-    "web3": "1.0.0-beta.37",
-    "web3-utils": "1.0.0-beta.37"
+    "web3": "1.2.4",
+    "web3-utils": "1.2.4"
```

Comment:
Why do we import web3-utils in wallet kit? It's included in web3 right? Nbd if you'd prefer to leave it as is though

Reply:
Not sure exactly why - think it is accessible by web3.utils, but I am just trying to minimize walletkit changes till we can deprecate